lib/elixir_console_web/templates/console/command_input.html.leex

```diff
@@ -7,7 +7,7 @@
       class="ml-2 bg-transparent flex-1 outline-none"
       autocomplete="off"
       name="command"
-      phx-keydown="suggest"
+      phx-window-keydown="suggest"
```

Comment:
We should not change this. If we leave the keyboard event attached to the window it is kind of global. I tested it and it feels awkward to hit TAB when the focus is not in the console text input but still showing autocomplete options (if the text is actually autocompleted, when there is only one possibility, the focus returns to the input).

Reply:
Good catch! This part was very confusing in the docs, in some versions it replaces where it said phx-keydown for phx-window-keydown, but in parts of the document it still refers to phx-keydown. Now I see the difference between them and it makes sense to have it without the change. Thanks!